reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
+# Usage reset_security_server.sh -n=server(s) -h=path/to/hosts.txt -a=path/to/ansible
+# Usage example: reset_security_server.sh -n=ss3,ss4 -h=../../X-Road/ansible/hosts/lxd_hosts.txt -a=../../X-Road/ansible
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+    exit 1
+fi
+
+for i in "$@"
+do
+case $i in
+    -n=*|--name=*)
+    NAME="${i#*=}"
+    shift
+    ;;
+    -h=*|--hosts=*)
+    HOSTS="${i#*=}"
+    shift
+    ;;
+    -a=*|--ansible=*)
+    ANSIBLE="${i#*=}"
+    shift
+    ;;
+    *)
+        echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+        exit 1
+    ;;
+esac
+done
+
+names=$(echo "$NAME" | tr "," "\n")
+container_list=$(lxc list --format=json | jq -r '.[] | .name + ","')
```

Comment:
When using functions, then one can avoid global namespace pollution by defining function-local variables:

```
local: local [option] name[=value] ...
    Define local variables.
    
    Create a local variable called NAME, and give it VALUE.  OPTION can
    be any option accepted by `declare'.
    
    Local variables can only be used within a function; they are visible
    only to the function where they are defined and its children.
    
    Exit Status:
    Returns success unless an invalid option is supplied, a variable
    assignment error occurs, or the shell is not executing a function.
```

Reply:
The script now uses functions with parameters